### ISSUE TITLE
dataobtained

### ISSUE THREAD

**Issue by baijingtingfuren** [sergey-volokhin/TextGCN#1]
Hello, may I know where the data in the kg_readable.tsv file is obtained from? Thank you.

**Comment by sergey-volokhin** [comment]
@baijingtingfuren Hello. 
The reviews and metadata are from [Julian McAuley's](https://cseweb.ucsd.edu/~jmcauley/datasets/amazon_v2/) website, from the "per-category" table. 
I have also uploaded a [`data/process_data.py`](../blob/master/data/process_data.py) script that processes raw `.json` files and produces the `kg_readable.tsv, train.tsv, test.tsv` files, required to run the model. 
I hope that helps. Tell me if you have any other questions.

**Comment by baijingtingfuren** [comment]
@sergey-volokhin Hello, how did you handle the `meta_Toys_and_Games.json` part in the `Toys_and_Games dataset`(Because the dataset I downloaded contains a mixture of single and double quotation marks, it is difficult to clean.)? Also, where were `item_list.txt` and `user_list.txt` constructed? Lastly, what version of the **dgl** library are you using?
Your response has been very helpful to me. I wish you all the best.

**Comment by sergey-volokhin** [comment]
@baijingtingfuren I have updated the code and README file, it should be clearer now.
 
The same script [`process_data.py`](../blob/master/data/process_data.py) reads the metadata file one line at a time using json, and removes all the fields except the title, description, asin, and time. The strings are cleaned and normalized in the script, so you don't need to do anything manually. It then saves the processed reviews into `reviews_synced.tsv` file, processed metadata into `kg_readable.tsv` file, as well as splits the reviews into train and test files. 

The `item_list.txt` and `user_list.txt` files aren't needed anymore, they are part of a legacy way to load data, I've removed them from the folder and from the code. 

I have also updated the requirements file, it now shows the versions of libraries I have used. You might want to change the index paths to the appropriate cuda versions for your machine.

I hope that helps.
